app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -82,7 +82,9 @@ export const FormAudience = ({
     experiment!.languages.map((v) => "" + v.id!),
   );
 
-  const [isSticky, setIsSticky] = useState<boolean>(false);
+  const [isSticky, setIsSticky] = useState<boolean>(
+    experiment.isSticky ?? false,
+  );
```

Comment:
Oh right yeah, we missed this on the first pr 😅 Anyways looks good but can you pls also add a couple tests to verify the cases where it's set on the model or not 🙏